app/assets/stylesheets/elements/_dropdown.scss

```diff
@@ -38,6 +38,10 @@
   overflow-y: auto;
   z-index: 100;
 
+  @media (max-width: 999px) {
+    right: -70px;
```

Comment:
This breaks other dropdowns though
![image](https://github.com/Mitcheljager/workshop.codes/assets/6181929/a1fbed24-2c22-4f88-b727-bc5421f33ff7)

Reply:
ah good catch. i thought i checked all the dropdowns but i forgot the save dropdown 😩.
should i just add a new class like this? what would be a good name for it?

```css
.dropdown__adjust-right{
  @media (max-width: 999px) {
    right: -70px;
  }
}
```